docs/features/virtual_folders.rst

```diff
@@ -0,0 +1,69 @@
+.. _virtual_folders:
+
+Virtual Folders
+===============
+
+.. versionadded:: 2.7
+
+Virtual folders provide a way to group translations based on any criteria,
+including a same file across all the languages in a project, or specific files.
```

Comment:
It means that a file named `file.po` is matched in all languages in a project. For example `/af/proj/file.po` or `/eu/proj/file.po`.

Rephrased to avoid confusion.